shared/src/common/enums/exceptions/exception-message.enum.ts

```diff
@@ -6,6 +6,7 @@ enum ExceptionMessage {
   INVALID_GROUP_NAME = 'Group name is invalid.',
   INVALID_GROUP_PERMISSIONS = 'Group permissions are invalid.',
   INVALID_GROUP_USERS = 'Group users are invalid.',
+  PERMISSION_LACK = 'You don`t have permission to access this resource.',
```

Comment:
Let's just escape simple quote
```suggestion
  PERMISSION_LACK = 'You don\'t have permission to access this resource.',
```

Reply:
Prettier autocorrects it to double quotes, that triggers eslint rule (Strings must use singlequote.)